myuw/templates/handlebars/card/instructor_schedule/course_eval.html

```diff
@@ -0,0 +1,64 @@
+                <h4 class="myuw-card-row-heading">Evaluations</h4>
+            </div>
+
+            <div class="col-sm-9">
+              <div rel="{{curriculum_abbr}} {{course_number}} {{section_id}}">
+                <ul class="myuw-card-itemlist">
+
+                {{#if evaluation.eval_status}}
+                  {{#if evaluation.is_online}}
+                    {{#if evaluation.eval_open_date}}
+                      {{#if evaluation.is_open}}
+                        <li class="evaluation-sechedule">
+                            <span>Open:&nbsp;&nbsp;{{evaluation.response_rate_percent}}% of responses received</span>
+                            <p class="myuw-note">Closes {{evaluation.eval_close_date_display}}</p>
+                        </li>
+                        <li class="evaluation-results">
+                            <span>Results:&nbsp;&nbsp;Report will be available {{evaluation.report_available_date_display}}</span>
+                        </li>
+                        <li><a href="https://{{#if is_bothell}}uwb{{else}}{{#if is_tacoma}}uwt{{else}}uw{{/if}}{{/if}}.iasystem.org/faculty">Manage evaluation</a></li>
+                      {{else}}
+                        {{#if evaluation.is_past}}
+                            <li>
+                                <span>Closed {{evaluation.eval_close_date_display}}{{#if evaluation.response_rate_percent}} with a {{evaluation.response_rate_percent}}% response rate{{/if}}.</span>
+                            </li>
+                            {{#if evaluation.report_url}}
+                              <li>
+                                  <a href="{{evaluation.report_url}}">View evaluation results report</a>
+                              </li>
+                            {{/if}}
+                        {{else}} <!-- opening in the future -->
+                            <li class="evaluation-sechedule">Online evaluation will open {{evaluation.eval_open_date_display}}</li>
+                            <li><a href="https://{{#if is_bothell}}uwb{{else}}{{#if is_tacoma}}uwt{{else}}uw{{/if}}{{/if}}.iasystem.org/faculty">Manage evaluation</a></li>
+                        {{/if}}
+
+                      {{/if}}
+                    {{else}} <!-- no evaluation date, not scheduled -->
+                        {{#if evaluation.report_available_date}}
+                            <li class="evaluation-sechedule">Report Available {{evaluation.report_available_date_display}}</li>
+                        {{/if}}
+                        {{#if evaluation.report_url}}
+                          <li><a href="{{evaluation.report_url}}">View Report</a></li>
+                        {{/if}}
+                        <li class=""><strong>{{evaluation.response_rate_percent}}% Response rate</strong></li>
+                    {{/if}}
+
+                  {{else}}<!-- not online, paper based -->
+                      <li class="evaluation-sechedule">Paper-based evaluation</li>
```

Comment:
Is "sechedule" a misspelling of schedule? (also added on other list items above and below)